Cargo.toml

```diff
@@ -87,7 +87,7 @@ test-md-docs = ["electrum"]
 lazy_static = "1.4"
 env_logger = "0.7"
 clap = "2.33"
-electrsd = { version= "0.8", features = ["trigger", "bitcoind_0_21_1"] }
+electrsd = { version= "0.10", features = ["trigger", "bitcoind_0_21_1"] }
```

Comment:
With this upgrade I think you can get rid of extra cache lines in workflow too I think: https://github.com/bitcoindevkit/bdk/blob/master/.github/workflows/cont_integration.yml#L98